src/text/text_editor.py

```diff
@@ -285,14 +389,20 @@ async def show_dialog_as_float(dialog: PopUpDialog) -> None:
 
 # All the do_ is a menu item
 
-# TODO actually make new file
+
 def do_new_file() -> None:
-    """Doesn't make new file just clears text_field but i guess thats fine till save"""
+    """Makes a new file"""
     text_field.text = ""
+    set_current_path(None)
+    set_title("ThoughtBox - Untitled")
 
 
 def do_exit() -> None:
     """Exit"""
+    with open("user_setting.json", "w") as j:
```

Comment:
```suggestion
    with open(".user_setting.json", "w") as j:
```
Let's make this a hidden file.